spec/blocksync/verification.md

```diff
@@ -0,0 +1,120 @@
+## Block verification
+
+When blocksyncing, a node is not participating in conensus. It is receiving blocks that have already been decided and committed. To avoid being fooled by malicious peers, the node has to verify the received blocks before executing the transactions and storing the block in its store. 
+
+
+The verification in blocksync aims to apply the same logic as the [light client verification](../light-client/verification/README.md). The safety guarantees of the light client model are provided by verifiying blocks starting from an **initial trusted state** and using validators who were bonded within a **trusting period**. Once this period expires, the validators could have unbonded, and we cannot rely on them being hoenst anymore. In blocksync, we can be **outside** the trusting period. Therefore, the guarantees provided by blocksync verification can be divided in two groups: 
+
+- The block is within the trusting period, we can punish validators, and provide guarantees that if a block is signed by trusted validators, they are indeed correct. 
```

Comment:
The guarantees in this case should b the same as the ones provided by the light client - at least a subset of them is honest. As we do witness verification, we will catch the dishonest one because we will send evidence to the nodes to determine which one is lying.